.github/workflows/build-master-blue-arm64.yaml

```diff
@@ -62,10 +62,18 @@ jobs:
       - name: Run make deps_ci
         run: |
           export DOCKER_INSTALL=true
+          export DOCKER_INSTALL=false
+          export LUET_INSTALL_FROM_COS_REPO=false
+          export LUET_VERSION=0.32.0
           sudo -E make deps_ci
```

Comment:
shouldnt deps_ci call the get_luet script? Like it does it already no?

Reply:
yup it does, but then when we call make deps we overwrite it with the version coming from the repos.. it's not nice, we re-install it twice here

Reply:
reworked it slightly, at least now should be a bit more obvious

Reply:
and I broke it somehow :D

Reply:
oh I see, ci is calling `deps_ci` now. adapted it